src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -127,6 +132,12 @@ export default {
 			}
 			return false
 		},
+		hasSocialId() {
+			const jCal = this.contact.jCal.slice(0)
+			const socialId = jCal[1].filter(props => props[0] === 'x-socialprofile')
```

Comment:
You do not need to slice the array, filter does not change the original array :)